### ISSUE TITLE
IDEA: zerocomment

### ISSUE THREAD

**Issue by risoul** [HelloZeroNet/ZeroNet#180]
Jus an usage idea.

Today lot of sites are closing comments, and moderation exclude lot of them. It will became more and more difficult to post a critical point of view.
I think we need to allow everybody to comment everywhere on the web, without moderation, with or without anonymity to criticize (even a sponsered product) and express opinions, on a decentralized base.

The idea is something like Disqus, but based on zeronet.
A browser plugin open a right column on the webpage we want to comment and create a zeronet article asociated with the visited url.
Then everyone with the plugin can see this column with comments and comment too or vote for best comments.

**Comment by HelloZeroNet** [comment]
I think it would make the things more centralized, so it's better to have separate and independent data for every site.

**Comment by risoul** [comment]
Yes, I did not use the good example with Disqus... the goal is not to centralize, but to get a zeronet comments associated with each website.

Manual website.com (an existing site with closed comments) example:

- I create a zeronet site corresponding to website.com, based on zerotalk, named zerotalk-website.com
- The url website.com/page1 as closed comments. I create a new discussion in zerotalk corresponding to this url.
- Then people can comment for each url, using the zerotalk-website.com site.

A browser plugin could automate the site and discussion creations on visiting the url (if not already done). Then it add a right column into the browser and fetch data from other users to show comments by loading the corresponding discussion. Finally the user could add a new comment.
Decentralized P2P based commenting for everyone system is born. Just an idea, may be not a good one.

**Comment by unrealwill** [comment]
@risoul  : Sort of a decentralized "commentanything.com" upon zeronet ?

**Comment by risoul** [comment]
Yes exactly! I did not know commentanything. The same based on zeronet = no moderation at all, no centralization, no third party service.

**Comment by border0464111** [comment]
Could it be achieved with merger feature ?
What need to be done here with this issue ?
